flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/processor/utils/InputPriorityConflictResolverTest.java

```diff
@@ -30,19 +33,43 @@
 import org.apache.flink.table.planner.plan.nodes.exec.spec.DynamicTableSourceSpec;
 import org.apache.flink.table.types.logical.IntType;
 import org.apache.flink.table.types.logical.RowType;
+import org.apache.flink.testutils.junit.extensions.parameterized.Parameter;
+import org.apache.flink.testutils.junit.extensions.parameterized.ParameterizedTestExtension;
+import org.apache.flink.testutils.junit.extensions.parameterized.Parameters;
 
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.Optional;
 import java.util.function.Consumer;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for {@link InputPriorityConflictResolver}. */
+@ExtendWith(ParameterizedTestExtension.class)
 class InputPriorityConflictResolverTest {
 
-    @Test
+    @Parameter
+    public Tuple2<BatchShuffleMode, StreamExchangeMode> batchShuffleModeAndStreamExchangeMode;
+
+    @Parameters(name = "batchShuffleModeAndStreamExchangeMode={0}")
+    public static Collection<Tuple2<BatchShuffleMode, StreamExchangeMode>> parameters() {
+        return Arrays.asList(
+                Tuple2.of(BatchShuffleMode.ALL_EXCHANGES_BLOCKING, StreamExchangeMode.BATCH),
+                Tuple2.of(BatchShuffleMode.ALL_EXCHANGES_HYBRID_FULL, StreamExchangeMode.BATCH),
```

Comment:
I have one question: what is the purpose of this case test?

Reply:
Currently, The `StreamExchangeMode` and `BatchShuffleMode` are not strictly binded, so I added the tests to verify these two modes.